src/config.ts

```diff
@@ -0,0 +1,31 @@
+import {CONTRACT_VERSION} from "./contract/constants";
+import {ContractVersionValue} from "./contract/types";
+import {encodeString} from "./util/util";
+
+class TinymanJSSDKConfig {
```

Comment:
I thought we might need to expose other configurations in the future so I named is like this (rather than `TransactionSettings`), but we can discuss

Reply:
Yea I wasn't sure of what the name of this class should be but Config seems to be better